drivers/at86rf2xx/include/at86rf2xx_internal.h

```diff
@@ -90,7 +108,12 @@ uint8_t at86rf2xx_reg_read(const at86rf2xx_t *dev, uint8_t addr);
  * @param[in] addr      address of the register to write
  * @param[in] value     value to write to the given register
  */
+#ifdef MODULE_AT86RFR2
+void at86rf2xx_reg_write(const at86rf2xx_t *dev, volatile uint8_t *addr,
+                         const uint8_t value);
+#else
```

Comment:
I'd recommend to change the name of the functions to `at86rfr2_reg_*` for the sake of consistency, since the API is changed.

Reply:
This has to be done when rebasing i think. But good to be alerted.